viewer/db.js

```diff
@@ -573,18 +583,12 @@ exports.numberOfUsers = function(cb) {
 
 exports.deleteUser = function (name, cb) {
   delete internals.usersCache[name];
-  return internals.usersElasticSearchClient.delete({index: internals.usersPrefix + 'users', type: 'user', id: name, refresh: true}, (err) => {
-    delete internals.usersCache[name]; // Delete again after db says its done refreshing
-    cb(err);
-  });
+  return internals.usersElasticSearchClient.delete({index: internals.usersPrefix + 'users', type: 'user', id: name, refresh: true}, cb);
```

Comment:
i think you had a merge error here, its missing the new code

Reply:
sorry, I thought I fixed all those... will get on it